sdk/wallet-adapter/adapters/base-adapter/src/index.ts

```diff
@@ -27,12 +28,16 @@ export interface WalletAdapter {
     event: E,
     callback: WalletAdapterEvents[E]
   ) => () => void;
-  signTransaction(transaction: SignableTransaction): Promise<SignedTransaction>;
+  signTransaction(
```

Comment:
Lets instead move this to take an object, which can just be directly the `SuiSignTransactionInput`.